server/app/views/questiontypes/AddressQuestionRenderer.java

```diff
@@ -115,6 +115,14 @@ protected DivTag renderInputTags(
       cityField.forceAriaInvalid();
       stateField.forceAriaInvalid();
       zipField.forceAriaInvalid();
+      if (params
+          .errorDisplayMode()
+          .equals(ApplicantQuestionRendererParams.ErrorDisplayMode.DISPLAY_SINGLE_ERROR)) {
+        streetAddressField.focusOnError();
```

Comment:
question (blocking): what happens when you add "focus" to all of the fields? i tried it out and it focused on the first field for me (streetAddressField) even when I had filled that one out. I'm curious if we need to add `focusOnError` to all of the fields, or maybe just the first one. (apologies if we talked about this yesterday and I am not remembering!)

Reply:
Ah gotcha, that makes sense! I'm thinking about what would be the most clear for another dev coming across this. The multiple `focusOnError` calls would make me think that functionality is already in place. But it would also be confusing to only have the focus on the first one. I defer to your judgement, but what do you think about adding a comment to explain?

Reply:
Sounds good!